Library/Homebrew/cmd/update-report.rb

```diff
@@ -250,10 +250,23 @@ def migrate_tap_migration
       tabs = dir.subdirs.map { |d| Tab.for_keg(Keg.new(d)) }
       next unless tabs.first.tap == tap # skip if installed formula is not from this tap.
       new_tap = Tap.fetch(new_tap_name)
-      new_tap.install unless new_tap.installed?
-      # update tap for each Tab
-      tabs.each { |tab| tab.tap = new_tap }
-      tabs.each(&:write)
+      # related to formulae deduplication. If formula stays in cask, autoinstall or provide cask instructions
```

Comment:
`related to formulae deduplication.` sounds like a comment that is relevant now, but won't make any sense if this was looked at in a few months. Maybe the whole comment can be rephrased and also moved into the block following `if new_tap_name == "caskroom/cask"`? Here's as suggestion:

> For formula migrated to cask: Auto-install cask or provide install instructions.